bin/single-new-eks-inferentia-opensource-observability.ts

```diff
@@ -0,0 +1,6 @@
+import SingleNewEksInferentiaOpenSourceObservabilityPattern from '../lib/single-new-eks-opensource-observability-pattern/neuron/inferentia-index';
+import { configureApp } from '../lib/common/construct-utils';
+
+const app = configureApp();
+
+new SingleNewEksInferentiaOpenSourceObservabilityPattern(app, 'single-new-eks-inferentia-opensource');
```

Comment:
Update the `readMe.md` `make list` command output with this new pattern.

Reply:
it is there